src/faq.md

```diff
@@ -79,7 +79,62 @@ outputs:
       outputEval: ${self[0].basename=inputs.otu_table_name; return self;}
 ```
 
-## Setting `self`-based input bindings for optional inputs
+## Referencing a Local Script
+
+There are two ways to reference a local script:
+
+The first method involves adding the folder containing your scripts to the `PATH` environment variable. This allows you to run the shell script without using `sh` or `bash` commands.
+
+Start with adding a _shebang_ at the top of your file:
+
+```{code-block}
+#!/bin/bash
+```
+
+After that, make the script executable with the command `chmod +x scriptname.sh`
+
+Finally, modify your `PATH` to add the directory where your script is located.
+(It is good practice to use `$HOME/bin` for storing your own scripts).
+
+```bash
+export PATH=$PATH:/appropriate/directory
+```
+
+Now you can use `baseCommand: scriptname.sh` to run the script directly.
+
+```cwl
+#!/bin/bash
+cwlVersion: v1.0
```

Comment:
Again, if we except people to follow a particular formatting, then it should be consistent in the User Guide so people know what to do.  I recommend for things like this - you make a list, change them in the user guide and then make sure people can find the list,  perhaps in the contributions.md.  Also, not sure how @tetron or I would have know about what you wanted since it is inconsistent in the User Guide that you approved.

Reply:
This was an instruction that Bruno followed, but we didn't document elsewhere:

https://github.com/common-workflow-language/user_guide/issues/222#issuecomment-1172865593

Issue for follow-up: https://github.com/common-workflow-language/user_guide/issues/349